Misc/NEWS.d/next/Core and Builtins/2018-01-19-01-54-22.bpo-31368.kzKqUR.rst

```diff
@@ -0,0 +1,2 @@
+Expose preadv (preadv2) and pwritev (pwritev2) system calls in the os module.
```

Comment:
I suggest: Add os.preadv() and os.pwritev() functions.

Again, I don't think that it's needed to mention the underlying C functions.